src/modules/display.js

```diff
@@ -0,0 +1,36 @@
+import get from './get';
+import post from './post';
+
+const nameInput = document.querySelector('.name');
+const scoreInput = document.querySelector('.score');
+const tbody = document.querySelector('.tbody');
+const submitBtn = document.querySelector('.submit');
+const refreshbtn = document.querySelector('.refresh');
+
+const display = async () => {
+  const names = await get();
+  //   console.log(names);
```

Comment:
- Kindly remove commented code, it's unnecessary to push comments like this to your repository